document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapping/smallfull/model/SmallFullApiData.java

```diff
@@ -101,30 +103,52 @@ public void setCreditorsWithinOneYear(CreditorsWithinOneYearApi creditorsWithinO
       this.creditorsWithinOneYear = creditorsWithinOneYear;
     }
 
+    public EmployeesApi getEmployees() {
+        return employees;
+    }
+
+    public void setEmployees(EmployeesApi employees) {
+        this.employees = employees;
+    }
+
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (!(o instanceof SmallFullApiData)) return false;
+        if (this == o)
+            return true;
+        if (! (o instanceof SmallFullApiData))
+            return false;
         SmallFullApiData that = (SmallFullApiData) o;
         return Objects.equals(getCurrentPeriod(), that.getCurrentPeriod()) &&
                 Objects.equals(getPreviousPeriod(), that.getPreviousPeriod()) &&
                 Objects.equals(getCompanyProfile(), that.getCompanyProfile()) &&
                 Objects.equals(getApproval(), that.getApproval()) &&
                 Objects.equals(getBalanceSheetStatements(), that.getBalanceSheetStatements()) &&
                 Objects.equals(getAccountingPolicies(), that.getAccountingPolicies()) &&
+                Objects.equals(getTangibleAssets(), that.getTangibleAssets()) &&
                 Objects.equals(getDebtors(), that.getDebtors()) &&
-                Objects.equals(getCreditorsWithinOneYear(), that.getCreditorsWithinOneYear()) &&
-                Objects.equals(getTangibleAssets(), that.getTangibleAssets());
+                Objects.equals(getEmployees(), that.getEmployees()) &&
+                Objects.equals(getCreditorsWithinOneYear(), that.getCreditorsWithinOneYear());
     }
 
     @Override
     public int hashCode() {
-      return Objects.hash(accountingPolicies, approval, balanceSheetStatements, companyProfile,
-          creditorsWithinOneYear, currentPeriod, debtors, previousPeriod, tangibleAssets);
+        return Objects.hash(getCurrentPeriod(), getPreviousPeriod(), getCompanyProfile(),
+                getApproval(), getBalanceSheetStatements(), getAccountingPolicies(), getTangibleAssets(), getDebtors(), getEmployees(), getCreditorsWithinOneYear());
     }
-    
+
     @Override
     public String toString() {
-        return new Gson().toJson(this);
```

Comment:
What's the motivation behind removing `Gson`?

Reply:
Auto generated toString method and it seems to have removed the gson